src/t_mockiato.rs

```diff
@@ -0,0 +1,367 @@
+        }
+
+        let _mock = AMock::new();
+    }
+
+    fn multi_trait() {
+        // Trait bounds are not supported, and custom derive is the only way to
+        // mock a trait, so there's no way for a mock to implement multiple
+        // traits.
+        unimplemented!()
+    }
+
+    fn return_call_with_args() {
+        unimplemented!()
+    }
+
+    fn return_reference() {
+        // Mockiato can only return 'static references
+        unimplemented!()
+    }
+
+    fn return_mutable_reference() {
+        // Mockiato can only return 'static references
+        unimplemented!()
+    }
+
+    fn return_owned() {
+        #[mockable]
+        pub trait A {
+            fn foo(&self) -> UniquelyOwned;
+        }
+
+        let mut mock = AMock::new();
+        let result = UniquelyOwned(42);
+        mock.expect_foo()
+            .returns_once(result);
+        assert_eq!(mock.foo(), UniquelyOwned(42));
+    }
+
+    fn return_parameters() {
+        unimplemented!()
+    }
+
+    fn send() {
+        #[mockable]
+        pub trait A {}
+
+        let mock = AMock::new();
+        let _ = Box::new(mock) as Box<A + Send>;
+    }
+
+    fn static_method() {
+        // The first parameter of a method must be self, so that the trait is
+        // object-safe
+        unimplemented!()
+    }
+
+    fn times_range() {
+        #[mockable]
+        pub trait A {
+            fn foo(&self);
+        }
+        let mut mock = AMock::new();
+        mock.expect_foo().times(2..4).returns(());
+        mock.foo();
+        mock.foo();
+    }
+
+    fn derive() {
+        #[mockable]
+        pub trait A {
+            fn foo(&self, key: i16);
+        }
+
+        let _mock = AMock::new();
+    }
+
+    fn fallback() {
+        unimplemented!()
+    }
+
+    fn match_combo() {
+        unimplemented!()
+    }
+
+    fn match_constant() {
+        #[mockable]
+        pub trait A {
+            fn foo(&self, key: i16);
+        }
+
+        let mut mock = AMock::new();
+        mock.expect_foo(|key| key.partial_eq(5));
+        mock.foo(5);
+    }
+
+    fn match_operator() {
+        unimplemented!()
+    }
+
+    fn match_pattern() {
+        unimplemented!()
+    }
+
+    fn match_range() {
+        unimplemented!()
+    }
+
+    fn match_wildcard() {
+        #[mockable]
+        pub trait A {
+            fn foo(&self, key: i16);
+        }
+
+        let mut mock = AMock::new();
+        mock.expect_foo(|key| key.any());
+        mock.foo(2);
+    }
+
+    fn modules() {
+        // Only traits can be mocked
+        unimplemented!()
+    }
+
+    fn return_constant() {
+        #[mockable]
+        pub trait A {
+            fn foo(&self) -> i16;
+        }
+
+        let mut mock = AMock::new();
+        mock.expect_foo().returns(2i16);
+        assert_eq!(mock.foo(), 2);
+    }
+
+    // Mockiato implemented this feature only for one type: ()
+    fn return_default() {
+        unimplemented!()
+    }
+
+    fn return_panic() {
```

Comment:
This is [supported](https://docs.rs/mockiato/0.9.0/mockiato/struct.MethodCallBuilder.html#method.panics):
```rust
let mut mock = AMock::new();
mock.expect_foo()
    .panics_with_message("Some panic message")
    .times(..);
```

Reply:
Ok.  I didn't notice that in the docs, but I see it now.